packages/services-store/src/actions/profileAPI.ts

```diff
@@ -16,13 +16,25 @@ import { UserStatus, PROFILE_API_ACTION, ProfileAPIState } from '../types/profil
  * @returns A Redux action to set the state that the JSONP call for user authentication status failed.
  * @private
  */
-export function setMonitorUserStatusError(error: Error) {
+export function setErrorRequestUserStatus(error: Error) {
   return {
-    type: PROFILE_API_ACTION.SET_ERROR_MONITOR_USER_STATUS,
+    type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
     error,
   };
 }
 
+/**
+ * @param status The promise of the REST call for  user status that is in progress.
+ * @returns A Redux action to set the state that the REST call for user status is in progress.
+ * @private
+ */
+export function setRequestUserStatusInProgress(status: Promise<UserStatus>) {
+  return {
+    type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+    status,
+  };
+}
```

Comment:
As seen at https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.14.0/packages/services-store/src/actions/localeAPI.ts#L14-L24, we use `request` variable/property names (instead of `status`) in this kind of Redux action. There are three places to make the change, one is JSDoc comment, one is function arg, one is property name.